08-communicating-results.Rmd

```diff
@@ -44,66 +49,64 @@ anes_des <- anes_adjwgt %>%
 
-> As of [DATE], an estimated XX% of registered U.S. voters say they will vote for [CANDITATE NAME] for president in the [YEAR] general election.
+> As of [DATE], an estimated XX% of registered U.S. voters say they will vote for [CANDIDATE NAME] for president in the [YEAR] general election.
 
-This sentence provides a few key pieces of information for the audience:
+This sentence provides key pieces of information in a straightforward way:
 
- 1. **[DATE]**: Given that polling data is dependent on a point in time, providing the date of reference is important for understanding when this data is valid.
- 2. **Registered U.S. voters**: This is the target population, and by including this information, we are telling the audience the population for reference and who was surveyed.
- 3. **[CANDITATE NAME] for president**: This provides the information on the estimate. The number is the percentage of those voting for a specific candidate for a specific office.
- 4. **[YEAR] general election**: As with the bullet above, this information provides more context around the specific election and year. The estimate would take on a different meaning if we changed it to a *primary* election instead of a *general* election, for example.
+ 1. **[DATE]**: Given that polling data is time-specific, providing the date of reference lets the audience know when this data was valid.
+ 2. **Registered U.S. voters**: This tells the audience who we surveyed, letting them know the target population.
+ 3. **XX%**: This part provides the estimated percentage of people voting for a specific candidate for a specific office.
+ 4. **[YEAR] general election**: As with the bullet above, adding this gives more context about the election type and year. The estimate would take on a different meaning if we changed it to a *primary* election instead of a *general* election.
  
-This sentence also includes the word "estimated." When presenting results in aggregate from surveys, it is important not to talk about estimates in the absolute as we have errors around each estimate. Using words like "estimated," "on average," or "around" will help convey the uncertainty with a given value. Including that uncertainty would be more informative to the audience. For example, a sentence could include uncertainty with a margin of error such as "XX% (+/- Y%)". Confidence intervals can also be incorporated into the text to assist readers.  
+We also included the word "estimated." When presenting aggregate survey results, we have errors around each estimate. We want to convey this uncertainty rather than talk in absolutes. Words like "estimated," "on average," or "around" can help our communication to the audience. Instead of saying 'XX%,' we can say 'XX% (+/- Y%)' to show the margin of error. Confidence intervals can also be incorporated into the text to assist readers.  
```

Comment:
```suggestion
We also included the word "estimated." When presenting aggregate survey results, we have errors around each estimate. We want to convey this uncertainty rather than talk in absolutes. Words like "estimated," "on average," or "around" can help communicate this uncertainty to the audience. Instead of saying 'XX%,' we can also say 'XX% (+/- Y%)' to show the margin of error. Confidence intervals can also be incorporated into the text to assist readers.  
```